elide-model-config/src/main/java/com/yahoo/elide/modelconfig/parser/handlebars/HandlebarsHelper.java

```diff
@@ -141,6 +142,8 @@ public String getJavaType(Type type) {
                 return BIGDECIMAL;
             case MONEY:
                 return BIGDECIMAL;
+            case TIME:
+                return DATE;
```

Comment:
if we return "Time" instead of "Date", we can remove if-else block for dimension type in table.hbs file.
Otherwise this is unused and can be removed.